deploy/009-deploy-vtokens.ts

```diff
@@ -110,7 +114,11 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
       const VToken = await ethers.getContractFactory("VToken");
       const underlyingDecimals = Number(await tokenContract.decimals());
       const vTokenDecimals = 8;
-      const treasuryAddress = await toAddress(preconfiguredAddresses.VTreasury || "VTreasury", hre);
+      const protocolShareReserveAddress = await toAddress(
+        preconfiguredAddresses.ProtocolShareReserve || "ProtocolShareReserve",
+        hre,
+      );
```

Comment:
Instead of increase the number of imports and "hardcoding" addresses, to get the PSR address, we could follow the same approach we followed [here](https://github.com/VenusProtocol/isolated-pools/blob/develop/deploy/015-funds-config.ts#L40-L51):

1) try to get the PSR address
2) if it doesn't exist and we are on hardhat, deploy it

(we can encapsulate this in a function)

This would allow us to remove the `preconfiguredAddresses.ProtocolShareReserve` entries from the `deploymentConfig.ts` file